docs/01_nodeos/03_plugins/producer_plugin/index.md

```diff
@@ -106,6 +107,21 @@ Config Options for eosio::producer_plugin:
 
 * [`chain_plugin`](../chain_plugin/index.md)
 
+## The priority of transaction
+
+You can give one of the transaction types priority over another when the producer plugin has a queue of transactions pending.
+
+The option below sets the ratio between the incoming transaction and the deferred transaction:
+
+```console
+  --incoming-defer-ratio arg (=1)       
+```
+
+By default value of `1`, the `producer` plugin processes one incoming transaction per each deferred transaction. When `arg` sets to `10`, the `producer` plugin processes 10 incoming transactions per each deferred transaction. 
```

Comment:
Suggestion to remove the `per` and `each` redundancy:

By default value of `1`, the `producer` plugin processes one incoming transaction per ~~each~~ deferred transaction. When `arg` value is set to `10`, the `producer` plugin processes 10 incoming transactions per ~~each~~ deferred transaction.